### ISSUE TITLE
New installation 500 error

### ISSUE THREAD

**Issue by gmorken** [keeps/roda#2231]
Hi
I have installed roda in an azure container instance. Problem is that I am getting 500 server error when navigating to the frontpage.
Error in console log in browser is
The version is keeps/roda:latest

Failed to load resource: the server responded with a status of 500 () :8080/gwtrpc/userlogin:1 
          
        
 org.roda.wui.client.main.Menu
     
SEVERE: Error getting Authenticated user
com.google.gwt.user.client.rpc.StatusCodeException: 500  The call failed on the server; see server log for details
	at Unknown.Gg(org.roda.wui.RodaWUI-0.js)
	at Unknown.Ug(org.roda.wui.RodaWUI-0.js)
	at Unknown.lZb(org.roda.wui.RodaWUI-0.js)
	at Unknown.new GZb(org.roda.wui.RodaWUI-0.js)
	at Unknown.I1b(org.roda.wui.RodaWUI-0.js)
	at Unknown.pC(org.roda.wui.RodaWUI-0.js)
	at Unknown.EC(org.roda.wui.RodaWUI-0.js)
	at Unknown.eval(org.roda.wui.RodaWUI-0.js)
	at Unknown.Sq(org.roda.wui.RodaWUI-0.js)
	at Unknown.Vq(org.roda.wui.RodaWUI-0.js)
	at Unknown.eval(org.roda.wui.RodaWUI-0.js)

_Kb @ org.roda.wui.RodaWUI-0.js:9654

**Comment by luis100** [comment]
You need to get into the container and find the log file in /roda/log/roda-core.log
There is probably details about an initialization error there.

**Comment by gmorken** [comment]
Ok thanks.
Here is the log:

2022-09-05 10:40:25,293 [localhost-startStop-1] DEBUG org.roda.core.RodaCoreFactory - RODA HOME is /roda
2022-09-05 10:40:25,993 [localhost-startStop-1] INFO  org.roda.core.RodaCoreFactory - Copying files from classpath prefix=config/, destination=/roda/example-config, removePrefix=true, excludePaths=[config/ldap, config/i18n/client, config/i18n/I18N.gwt.xml]
2022-09-05 10:40:27,207 [localhost-startStop-1] DEBUG org.roda.core.RodaCoreFactory - Finished instantiating essential directories
2022-09-05 10:40:27,714 [localhost-startStop-1] DEBUG org.roda.core.RodaCoreFactory - Loaded roda-core.properties
2022-09-05 10:40:27,814 [localhost-startStop-1] DEBUG org.roda.core.RodaCoreFactory - Loaded roda-roles.properties
2022-09-05 10:40:27,843 [localhost-startStop-1] DEBUG org.roda.core.RodaCoreFactory - Loaded roda-permissions.properties
2022-09-05 10:40:27,877 [localhost-startStop-1] DEBUG org.roda.core.RodaCoreFactory - Going to instantiate Filesystem on '/roda/data/storage'
2022-09-05 10:40:27,901 [localhost-startStop-1] WARN  o.r.c.storage.fs.FileStorageService - Trying to initialize /roda/data/storage-history/data/aip but parent is not writable
2022-09-05 10:40:27,901 [localhost-startStop-1] WARN  o.r.c.storage.fs.FileStorageService - Trying to initialize /roda/data/storage-history/metadata/aip but parent is not writable
2022-09-05 10:40:27,905 [localhost-startStop-1] DEBUG org.roda.core.RodaCoreFactory - Finished instantiating storage...
2022-09-05 10:40:27,994 [localhost-startStop-1] DEBUG org.roda.core.RodaCoreFactory - Finished instantiating model...
2022-09-05 10:40:27,994 [localhost-startStop-1] DEBUG org.roda.core.RodaCoreFactory - Finished instantiating storage & model
2022-09-05 10:40:27,994 [localhost-startStop-1] INFO  org.roda.core.RodaCoreFactory - Using SOLR home: /usr/local/tomcat/temp/rodaWorkingDirectory242549814398412265/index6755683151718187943/config/index
2022-09-05 10:40:28,049 [localhost-startStop-1] INFO  org.roda.core.RodaCoreFactory - Copying files from classpath prefix=config/index/common/conf/, destination=/usr/local/tomcat/temp/solr-base-config8055876308357204916, removePrefix=true, excludePaths=[]
2022-09-05 10:40:28,470 [localhost-startStop-1] INFO  org.roda.core.RodaCoreFactory - Instantiating SOLR Embedded
2022-09-05 10:41:16,356 [http-nio-8080-exec-1] INFO  org.roda.wui.filter.OnOffFilter - CASSingleSignOutFilter is OFF
2022-09-05 10:41:16,368 [http-nio-8080-exec-1] INFO  org.roda.wui.filter.OnOffFilter - CASValidationFilter is OFF
2022-09-05 10:41:16,368 [http-nio-8080-exec-1] INFO  org.roda.wui.filter.OnOffFilter - CASRequestWrapperFilter is OFF
2022-09-05 10:41:16,504 [http-nio-8080-exec-7] INFO  org.roda.wui.filter.OnOffFilter - InternalApiAuthFilter is OFF
2022-09-05 10:41:16,505 [http-nio-8080-exec-7] INFO  org.roda.wui.filter.OnOffFilter - CasApiAuthFilter is OFF
2022-09-05 10:41:19,547 [http-nio-8080-exec-4] ERROR org.roda.wui.client.main.Menu - [10.92.0.6] Error getting Authenticated user, error: [com.google.gwt.user.client.rpc.StatusCodeException] 500  The call failed on the server; see server log for details
Unknown.Gg(org.roda.wui.RodaWUI-0.js)
Unknown.Ug(org.roda.wui.RodaWUI-0.js)
Unknown.lZb(org.roda.wui.RodaWUI-0.js)
Unknown.new GZb(org.roda.wui.RodaWUI-0.js)
Unknown.I1b(org.roda.wui.RodaWUI-0.js)
Unknown.pC(org.roda.wui.RodaWUI-0.js)
Unknown.EC(org.roda.wui.RodaWUI-0.js)
Unknown.eval(org.roda.wui.RodaWUI-0.js)
Unknown.Sq(org.roda.wui.RodaWUI-0.js)
Unknown.Vq(org.roda.wui.RodaWUI-0.js)
Unknown.eval(org.roda.wui.RodaWUI-0.js)
2022-09-05 10:41:44,643 [http-nio-8080-exec-4] ERROR org.roda.wui.servlets.ErrorHandler - [405] /gwtrpc/userlogin: HTTP method GET is not supported by this URL
2022-09-05 10:48:20,920 [http-nio-8080-exec-9] ERROR org.roda.wui.client.main.Menu - [10.92.0.5] Error getting Authenticated user, error: [com.google.gwt.user.client.rpc.StatusCodeException] 500  The call failed on the server; see server log for details
Unknown.Gg(org.roda.wui.RodaWUI-0.js)
Unknown.Ug(org.roda.wui.RodaWUI-0.js)
Unknown.lZb(org.roda.wui.RodaWUI-0.js)
Unknown.new GZb(org.roda.wui.RodaWUI-0.js)
Unknown.I1b(org.roda.wui.RodaWUI-0.js)
Unknown.pC(org.roda.wui.RodaWUI-0.js)
Unknown.EC(org.roda.wui.RodaWUI-0.js)
Unknown.eval(org.roda.wui.RodaWUI-0.js)
Unknown.Sq(org.roda.wui.RodaWUI-0.js)
Unknown.Vq(org.roda.wui.RodaWUI-0.js)
Unknown.eval(org.roda.wui.RodaWUI-0.js)
2022-09-05 10:48:58,342 [http-nio-8080-exec-6] ERROR org.roda.wui.client.main.Menu - [10.92.0.6] Error getting Authenticated user, error: [com.google.gwt.user.client.rpc.StatusCodeException] 500  The call failed on the server; see server log for details
Unknown.Gg(org.roda.wui.RodaWUI-0.js)
Unknown.Ug(org.roda.wui.RodaWUI-0.js)
Unknown.lZb(org.roda.wui.RodaWUI-0.js)
Unknown.new GZb(org.roda.wui.RodaWUI-0.js)
Unknown.I1b(org.roda.wui.RodaWUI-0.js)
Unknown.pC(org.roda.wui.RodaWUI-0.js)
Unknown.EC(org.roda.wui.RodaWUI-0.js)
Unknown.eval(org.roda.wui.RodaWUI-0.js)
Unknown.Sq(org.roda.wui.RodaWUI-0.js)
Unknown.Vq(org.roda.wui.RodaWUI-0.js)
Unknown.eval(org.roda.wui.RodaWUI-0.js)
2022-09-05 10:48:58,472 [http-nio-8080-exec-8] ERROR o.r.w.s.browse.BrowserServiceImpl - Unexpected error in find
java.lang.NullPointerException: null
        at org.roda.core.common.UserUtility.getGuest(UserUtility.java:239)
        at org.roda.core.common.UserUtility.getUser(UserUtility.java:84)
        at org.roda.core.common.UserUtility.getUser(UserUtility.java:96)
        at org.roda.wui.server.browse.BrowserServiceImpl.find(BrowserServiceImpl.java:321)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at com.google.gwt.user.server.rpc.RPC.invokeAndEncodeResponse(RPC.java:587)
        at com.google.gwt.user.server.rpc.RemoteServiceServlet.processCall(RemoteServiceServlet.java:333)
        at com.google.gwt.user.server.rpc.RemoteServiceServlet.processCall(RemoteServiceServlet.java:303)
        at com.google.gwt.user.server.rpc.RemoteServiceServlet.processPost(RemoteServiceServlet.java:373)
        at com.google.gwt.user.server.rpc.AbstractRemoteServiceServlet.doPost(AbstractRemoteServiceServlet.java:62)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:681)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
        at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
        at org.roda.wui.filter.OnOffFilter.doFilter(OnOffFilter.java:75)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
        at org.roda.wui.filter.OnOffFilter.doFilter(OnOffFilter.java:75)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
        at org.roda.wui.filter.OnOffFilter.doFilter(OnOffFilter.java:75)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
        at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
        at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
        at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:662)
        at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135)
        at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
        at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:698)
        at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
        at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:367)
        at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:639)
        at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
        at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:882)
        at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1691)
        at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
        at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
        at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
        at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
        at java.lang.Thread.run(Thread.java:750)
2022-09-05 10:48:58,600 [http-nio-8080-exec-10] ERROR o.r.w.c.c.utils.AsyncCallbackUtils - [10.92.0.6] Async request error, error: [org.roda.core.data.exceptions.GenericException] java.lang.NullPointerException
Unknown.Fg(org.roda.wui.RodaWUI-0.js)
Unknown.Qg(org.roda.wui.RodaWUI-0.js)
Unknown.FLc(org.roda.wui.RodaWUI-0.js)
Unknown.new bMc(org.roda.wui.RodaWUI-0.js)
Unknown.GMc(org.roda.wui.RodaWUI-0.js)
Unknown.I2b(org.roda.wui.RodaWUI-0.js)
Unknown.E2b(org.roda.wui.RodaWUI-0.js)
Unknown._0b(org.roda.wui.RodaWUI-0.js)
Unknown.I1b(org.roda.wui.RodaWUI-0.js)
Unknown.pC(org.roda.wui.RodaWUI-0.js)
Unknown.EC(org.roda.wui.RodaWUI-0.js)
Unknown.eval(org.roda.wui.RodaWUI-0.js)
Unknown.Sq(org.roda.wui.RodaWUI-0.js)
Unknown.Vq(org.roda.wui.RodaWUI-0.js)
Unknown.eval(org.roda.wui.RodaWUI-0.js)
2022-09-05 10:49:12,279 [http-nio-8080-exec-1] ERROR o.r.w.s.browse.BrowserServiceImpl - Unexpected failure
com.google.gwt.user.server.rpc.UnexpectedException: Service method 'public abstract java.lang.String org.roda.wui.client.browse.BrowserService.createAIP(java.lang.String,java.lang.String) throws org.roda.core.data.exceptions.AuthorizationDeniedException,org.roda.core.data.exceptions.GenericException,org.roda.core.data.exceptions.NotFoundException,org.roda.core.data.exceptions.RequestNotValidException,org.roda.core.data.exceptions.AlreadyExistsException' threw an unexpected exception: java.lang.NullPointerException
        at com.google.gwt.user.server.rpc.RPC.encodeResponseForFailure(RPC.java:416)
        at com.google.gwt.user.server.rpc.RPC.invokeAndEncodeResponse(RPC.java:605)
        at com.google.gwt.user.server.rpc.RemoteServiceServlet.processCall(RemoteServiceServlet.java:333)
        at com.google.gwt.user.server.rpc.RemoteServiceServlet.processCall(RemoteServiceServlet.java:303)
        at com.google.gwt.user.server.rpc.RemoteServiceServlet.processPost(RemoteServiceServlet.java:373)
        at com.google.gwt.user.server.rpc.AbstractRemoteServiceServlet.doPost(AbstractRemoteServiceServlet.java:62)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:681)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
        at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
        at org.roda.wui.filter.OnOffFilter.doFilter(OnOffFilter.java:75)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
        at org.roda.wui.filter.OnOffFilter.doFilter(OnOffFilter.java:75)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
        at org.roda.wui.filter.OnOffFilter.doFilter(OnOffFilter.java:75)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
        at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
        at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
        at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:662)
        at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135)
        at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
        at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:698)
        at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
        at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:367)
        at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:639)
        at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
        at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:882)
        at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1691)
        at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
        at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
        at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
        at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
        at java.lang.Thread.run(Thread.java:750)
Caused by: java.lang.NullPointerException: null
        at org.roda.core.common.UserUtility.getGuest(UserUtility.java:239)
        at org.roda.core.common.UserUtility.getUser(UserUtility.java:84)
        at org.roda.core.common.UserUtility.getUser(UserUtility.java:96)
        at org.roda.wui.server.browse.BrowserServiceImpl.createAIP(BrowserServiceImpl.java:386)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at com.google.gwt.user.server.rpc.RPC.invokeAndEncodeResponse(RPC.java:587)
        ... 37 common frames omitted
2022-09-05 10:49:12,349 [http-nio-8080-exec-6] ERROR o.r.w.c.c.utils.AsyncCallbackUtils - [10.92.0.6] Async request error, error: [com.google.gwt.user.client.rpc.StatusCodeException] 500  The call failed on the server; see server log for details
Unknown.Gg(org.roda.wui.RodaWUI-0.js)
Unknown.Ug(org.roda.wui.RodaWUI-0.js)
Unknown.lZb(org.roda.wui.RodaWUI-0.js)
Unknown.new GZb(org.roda.wui.RodaWUI-0.js)
Unknown.I1b(org.roda.wui.RodaWUI-0.js)
Unknown.pC(org.roda.wui.RodaWUI-0.js)
Unknown.EC(org.roda.wui.RodaWUI-0.js)
Unknown.eval(org.roda.wui.RodaWUI-0.js)
Unknown.Sq(org.roda.wui.RodaWUI-0.js)
Unknown.Vq(org.roda.wui.RodaWUI-0.js)
Unknown.eval(org.roda.wui.RodaWUI-0.js)
2022-09-05 10:49:13,351 [http-nio-8080-exec-4] ERROR o.r.w.s.browse.BrowserServiceImpl - Unexpected error in find
java.lang.NullPointerException: null
        at org.roda.core.common.UserUtility.getGuest(UserUtility.java:239)
        at org.roda.core.common.UserUtility.getUser(UserUtility.java:84)
        at org.roda.core.common.UserUtility.getUser(UserUtility.java:96)
        at org.roda.wui.server.browse.BrowserServiceImpl.find(BrowserServiceImpl.java:321)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at com.google.gwt.user.server.rpc.RPC.invokeAndEncodeResponse(RPC.java:587)
        at com.google.gwt.user.server.rpc.RemoteServiceServlet.processCall(RemoteServiceServlet.java:333)
        at com.google.gwt.user.server.rpc.RemoteServiceServlet.processCall(RemoteServiceServlet.java:303)
        at com.google.gwt.user.server.rpc.RemoteServiceServlet.processPost(RemoteServiceServlet.java:373)
        at com.google.gwt.user.server.rpc.AbstractRemoteServiceServlet.doPost(AbstractRemoteServiceServlet.java:62)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:681)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
        at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
        at org.roda.wui.filter.OnOffFilter.doFilter(OnOffFilter.java:75)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
        at org.roda.wui.filter.OnOffFilter.doFilter(OnOffFilter.java:75)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
        at org.roda.wui.filter.OnOffFilter.doFilter(OnOffFilter.java:75)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
        at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
        at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
        at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:662)
        at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135)
        at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
        at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:698)
        at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
        at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:367)
        at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:639)
        at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
        at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:882)
        at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1691)
        at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
        at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
        at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
        at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
        at java.lang.Thread.run(Thread.java:750)
2022-09-05 10:49:13,418 [http-nio-8080-exec-7] ERROR o.r.w.c.c.utils.AsyncCallbackUtils - [10.92.0.6] Async request error, error: [org.roda.core.data.exceptions.GenericException] java.lang.NullPointerException
Unknown.Fg(org.roda.wui.RodaWUI-0.js)
Unknown.Qg(org.roda.wui.RodaWUI-0.js)
Unknown.FLc(org.roda.wui.RodaWUI-0.js)
Unknown.new bMc(org.roda.wui.RodaWUI-0.js)
Unknown.GMc(org.roda.wui.RodaWUI-0.js)
Unknown.I2b(org.roda.wui.RodaWUI-0.js)
Unknown.E2b(org.roda.wui.RodaWUI-0.js)
Unknown._0b(org.roda.wui.RodaWUI-0.js)
Unknown.I1b(org.roda.wui.RodaWUI-0.js)
Unknown.pC(org.roda.wui.RodaWUI-0.js)
Unknown.EC(org.roda.wui.RodaWUI-0.js)
Unknown.eval(org.roda.wui.RodaWUI-0.js)
Unknown.Sq(org.roda.wui.RodaWUI-0.js)
Unknown.Vq(org.roda.wui.RodaWUI-0.js)
Unknown.eval(org.roda.wui.RodaWUI-0.js)

**Comment by gmorken** [comment]
I just created a new azure container instance and mounted an azure file share volume to /roda/data but same error log

**Comment by luis100** [comment]
well, it seems that the roda data folder (/roda/data) is not writable for some reason, you could try to have a shell into the container and check if there are permission issues with /roda/data folder.

**Comment by gmorken** [comment]
ok I can check that but I see these errors
2022-09-05 10:41:44,643 [http-nio-8080-exec-4] ERROR org.roda.wui.servlets.ErrorHandler - [405] /gwtrpc/userlogin: HTTP method GET is not supported by this URL

I dont see that this error belongs to that the disk is not writable.

or this one
022-09-05 10:41:19,547 [http-nio-8080-exec-4] ERROR org.roda.wui.client.main.Menu - [10.92.0.6] Error getting Authenticated user, error: [com.google.gwt.user.client.rpc.StatusCodeException] 500 The call failed on the server; see server log for details

**Comment by gmorken** [comment]
do you know what command I can run to check permissions?

I have successfully created a file with:
az container exec --name rodademo --resource-group rg-earchive --exec-command "touch /roda/data/storage/aip/foo.txt"

**Comment by gmorken** [comment]
Hi!
I am using the latest docker image keeps/roda
Today I tried to create a new container instance with more CPU and Memory but same error now

I have create the ACI with
`az container create --resource-group rg-earchive --name superroda2 --image registry.hub.docker.com/keeps/roda --dns-name-label superroda2 --ports 8080 --azure-file-volume-account-name sarodafiles3 --azure-file-volume-account-key <THEKEY> --azure-file-volume-share-name roda-files --azure-file-volume-mount-path /roda/data/ --cpu 2 --memory 3.5`

The file share is on azure and is Transaction optimized tier. During roda initialization Roda is able to create the tree structure on the file share
![image](https://user-images.githubusercontent.com/5782041/189831784-c6120a08-88eb-4b67-a286-edaedcf169eb.png)



But when the container starts the same error occurs
2022-09-13 06:35:01,902 [localhost-startStop-1] DEBUG org.roda.core.RodaCoreFactory - RODA HOME is /roda
2022-09-13 06:35:02,742 [localhost-startStop-1] INFO  org.roda.core.RodaCoreFactory - Copying files from classpath prefix=config/, destination=/roda/example-config, removePrefix=true, excludePaths=[config/ldap, config/i18n/client, config/i18n/I18N.gwt.xml]
2022-09-13 06:35:03,564 [localhost-startStop-1] DEBUG org.roda.core.RodaCoreFactory - Finished instantiating essential directories
2022-09-13 06:35:03,702 [localhost-startStop-1] DEBUG org.roda.core.RodaCoreFactory - Loaded roda-core.properties
2022-09-13 06:35:03,791 [localhost-startStop-1] DEBUG org.roda.core.RodaCoreFactory - Loaded roda-roles.properties
2022-09-13 06:35:03,811 [localhost-startStop-1] DEBUG org.roda.core.RodaCoreFactory - Loaded roda-permissions.properties
2022-09-13 06:35:03,826 [localhost-startStop-1] DEBUG org.roda.core.RodaCoreFactory - Going to instantiate Filesystem on '/roda/data/storage'
2022-09-13 06:35:03,884 [localhost-startStop-1] WARN  o.r.c.storage.fs.FileStorageService - Trying to initialize /roda/data/storage-history/data/aip but parent is not writable
2022-09-13 06:35:03,887 [localhost-startStop-1] WARN  o.r.c.storage.fs.FileStorageService - Trying to initialize /roda/data/storage-history/metadata/aip but parent is not writable
2022-09-13 06:35:03,919 [localhost-startStop-1] DEBUG org.roda.core.RodaCoreFactory - Finished instantiating storage...
2022-09-13 06:35:04,400 [localhost-startStop-1] DEBUG org.roda.core.RodaCoreFactory - Finished instantiating model...
2022-09-13 06:35:04,400 [localhost-startStop-1] DEBUG org.roda.core.RodaCoreFactory - Finished instantiating storage & model
2022-09-13 06:35:04,422 [localhost-startStop-1] INFO  org.roda.core.RodaCoreFactory - Using SOLR home: /usr/local/tomcat/temp/rodaWorkingDirectory1503849224236868968/index6694710594209370246/config/index
2022-09-13 06:35:04,475 [localhost-startStop-1] INFO  org.roda.core.RodaCoreFactory - Copying files from classpath prefix=config/index/common/conf/, destination=/usr/local/tomcat/temp/solr-base-config98715953182187868, removePrefix=true, excludePaths=[]
2022-09-13 06:35:04,921 [localhost-startStop-1] INFO  org.roda.core.RodaCoreFactory - Instantiating SOLR Embedded
2022-09-13 06:38:10,419 [http-nio-8080-exec-7] INFO  org.roda.wui.filter.OnOffFilter - CASSingleSignOutFilter is OFF
2022-09-13 06:38:10,421 [http-nio-8080-exec-7] INFO  org.roda.wui.filter.OnOffFilter - CASValidationFilter is OFF
2022-09-13 06:38:10,421 [http-nio-8080-exec-7] INFO  org.roda.wui.filter.OnOffFilter - CASRequestWrapperFilter is OFF
2022-09-13 06:38:10,536 [http-nio-8080-exec-1] INFO  org.roda.wui.filter.OnOffFilter - InternalApiAuthFilter is OFF
2022-09-13 06:38:10,537 [http-nio-8080-exec-1] INFO  org.roda.wui.filter.OnOffFilter - CasApiAuthFilter is OFF
2022-09-13 06:38:12,951 [http-nio-8080-exec-1] ERROR org.roda.wui.client.main.Menu - [10.92.0.6] Error getting Authenticated user, error: [com.google.gwt.user.client.rpc.StatusCodeException] 500  The call failed on the server; see server log for details
Unknown.Gg(org.roda.wui.RodaWUI-0.js)
Unknown.Ug(org.roda.wui.RodaWUI-0.js)
Unknown.lZb(org.roda.wui.RodaWUI-0.js)
Unknown.new GZb(org.roda.wui.RodaWUI-0.js)
Unknown.I1b(org.roda.wui.RodaWUI-0.js)
Unknown.pC(org.roda.wui.RodaWUI-0.js)
Unknown.EC(org.roda.wui.RodaWUI-0.js)
Unknown.eval(org.roda.wui.RodaWUI-0.js)
Unknown.Sq(org.roda.wui.RodaWUI-0.js)
Unknown.Vq(org.roda.wui.RodaWUI-0.js)
Unknown.eval(org.roda.wui.RodaWUI-0.js)
2022-09-13 06:45:59,387 [http-nio-8080-exec-4] INFO  org.roda.wui.servlets.ErrorHandler - [404] /manager/html: The requested resource [/manager/html] is not available

**Comment by luis100** [comment]
Hello,

We don't have experience with Azure containers, but the main issue is the lack of write permissions in essential folders. I see you have mapped `--azure-file-volume-mount-path /roda/data/` and the error `Trying to initialize /roda/data/storage-history/data/aip but parent is not writable`so I guess the issue is that the user within the container (by default UID 0, root) does not have write permissions to the folders mounted. So options would be to change permissions of the folders mapped into RODA so they can be written by the internal user, or to change the internal user used by RODA (option only available in RODA commercial version, contact sales@keep.pt if interested).

**Comment by luis100** [comment]
TL;DR from #2243 on RODA 4.4.0 we have breaking changes, we had to upgrade from Solr 7 to 8 because Solr 7 has reached end-of-life. If your setup is using Solr 7 you must use a RODA version lower than 4.4.0, or alternatively upgrade to Solr 8 (recommended). Side note, RODA 5 (in development, to be released soon) will require Solr 9.

**Comment by diglesias26** [comment]
Hi Luis, 

thanks for your response.
I have just installed an "old" roda version successfully in azure.

Regards!

**Comment by gmorken** [comment]
ok nice @diglesias26 what version is that?

**Comment by diglesias26** [comment]
keeps/roda:v4.3.1 
As Luis said, this is an unsupported version because it uses solr 7. But by the moment is enough for my purposes.

**Comment by micopiira** [comment]
Just to let you know, I also tried to run the latest roda on a pretty much fresh Ubuntu server instance and could not get it running. roda-core.log did not seem to contain anything important other than that it could not connect to Solr.

**Comment by luis100** [comment]
@micopiira connecting to Solr is a requirement and RODA will refuse to startup without it. Also, Solr needs to be available as an external service. Check the [standalone deployment example](https://github.com/keeps/roda/tree/master/deploys/standalone) to see how to get it started.
